services/getstream/GetstreamService.js

```diff
@@ -1,4 +1,3 @@
-const stream = require('getstream');
 const InvariantError = require('../../exceptions/InvariantError');
 const {MAX_FEED_FETCH_LIMIT, GETSTREAM_RANKING_METHOD} = require('../../helpers/constants');
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the `stream` import seems to be a good change if it's not being used in this file. However, please ensure that there are no references to `stream` elsewhere in the code that might have been missed.
```diff
- const stream = require('getstream');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->